mne/viz/topomap.py

```diff
@@ -616,6 +697,13 @@ def plot_topomap(data, pos, vmin=None, vmax=None, cmap=None, sensors=True,
         Handle for a function that is called when the user selects a set of
         channels by rectangle selection (matplotlib ``RectangleSelector``). If
         None interactive selection is disabled. Defaults to None.
+    extrapolate : str
+        If 'head' extrapolate to the edges of the head circle (does not work
+        when `outlines='skirt'`. If 'local' (default) extrapolate only to
+        nearby points (approximately to points closer than median
+        inter-electrode distance).
```

Comment:
I think there is an API problem here. How do users get the behavior they currently get in `master` for `'skirt'` on your PR? One option would be to keep `'head'` valid for `skirt` mode. Then "head" means "based on the head circle" (which would work and make sense in either `outlines` mode) and `local` means in a small neighborhood around the sensors (wherever they are).